pkg/controller/kieapp/defaults/defaults.go

```diff
@@ -1307,6 +1335,13 @@ func checkJvmOnServer(server *api.KieServerSet) {
 	setJvmDefault(server.Jvm)
 }
 
+func checkKafkaExt(server *api.KieServerSet) {
+	if server.KafkaExt == nil {
```

Comment:
by doing this, the kafka object will always be set, it is something expected?
I think the JMS integration only gets executed if if the expected property is enabled, might be interesting apply something similar for kafka.

Reply:
ok